Aptoide/aptoide-client-v8
# [Bugfix/ASV-1014] Facebook errors: Download Complete vs Download_99percent

**What does this PR do?**

   Deletes "Download_99percent" event and adds the "Download Complete" event. Detailed info is in the ticket

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] Bar.kt

**How should this be manually tested?**

   Check if the "Download Complete" and "Editors_Choice_Download_Complete" are being sent to facebook when appropriate (explicit by their name).
  
**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-1014](https://aptoide.atlassian.net/browse/ASV-1014)

**Code Review Checklist**
